rust/private/rust.bzl

```diff
@@ -609,6 +609,13 @@ _common_attrs = {
     "_error_format": attr.label(
         default = Label("//:error_format"),
     ),
+    "_experimental_toolchain_generated_sysroot": attr.label(
```

Comment:
Normally we put these flags on `rust_toolchain`. Any reason not to follow that pattern here? It would allow you to avoid adding it to two rules.

Reply:
Ahh good point. PTAL https://github.com/bazelbuild/rules_rust/commit/f8027f2e0ed683b9eabf3364fd43bc17ab162e13